pom.xml

```diff
@@ -1135,6 +1137,19 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.jvnet.jaxb2.maven2</groupId>
```

Comment:
this seems to be a plugin developed not under Apache, could you check? Do we need to add the plugin ? or just the jaxb-runtime ?